packages/manager/modules/pci/src/projects/project/storages/databases/database/current-queries/current-queries.html

```diff
@@ -123,5 +123,15 @@ <h2 data-translate="pci_databases_current_queries_title"></h2>
         >
             <span data-ng-bind="$value ? $value : '-'"></span>
         </oui-datagrid-column>
+        <oui-action-menu data-compact data-placement="end">
+            <oui-action-menu-item
+                data-on-click="$ctrl.trackAndTerminateQuery($row)"
+                data-disabled="$row.state.includes($ctrl.IDLE_QUERY_STATES[0])"
```

Comment:
I think that `$row.state` is a string and not an array, so I'm not sure you can use the `includes` function here.
You could either check that 
- `$row.state === $ctrl.IDLE_QUERY_STATES[0]` 
- or `$ctrl.IDLE_QUERY_STATES.includes($row.state)`
Depending on what statuses should disable the button

Reply:
Fixed